document/FileFormat.mdk

```diff
@@ -2672,16 +2675,20 @@ This can be used, for example, to designate the target of a jump instruction, or
 * `"declaration"`: The location introduces into the program a name which denotes an entity such as a variable, function, template, _etc_.
 
 * `"functionEnter"`: This location is an entry point to a function or method.
+    Every `annotatedCodeLocation` whose `kind` property is `"functionEnter"` shall be paired with a
+    subsequent `annotatedCodeLocation` whose `kind` property is `"functionExit"` and which refers to the same function,
+    unless the `codeFlow` in which the call occurs terminates before a function exit point is reached. 
 
-* `"functionExit"`: This location is an exit point from a function or method (for example, a return statement).
+* `"functionExit"`: This location represents the actual exit point from the function.
 
-* `"sanitizer"`: This is the location of a statement (for example, a function call), after the execution of which data that entered the system from outside (for example, from user input)
-is presumed to be safe.
-
-* `"sink"`: At this location, untrusted data enters some security-sensitive code (for example, an `eval` statement
-that converts untrusted text to executable code).
+* `"functionReturn"`: This is the location of a statement that returns control from a function or method
+(for example, a `return` statement).
 
-* `"source"`: At this location, untrusted data enters the system (for example, by being provided by a user or read from a file on disk).
+~ IsoNumberedNote
+Unlike `functionReturn`, `functionExit` is not associated with a source code location. It is a conceptual location,
```

Comment:
Fixed. I moved this description into the (normative) part of the description of `functionExit`.

---
In reply to: [77242895](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77242895) [](ancestors = 77242895)